### PR TITLE
QOL-8594 fix missing bracket and add tests to catch it in future

### PR DESCRIPTION
**Pull request by ThrawnCA**
Also combine similar tests into scenario outlines.

### CODE REVIEW

File: test/features/groups.feature [hunk 1]
```diff
@@ -1,53 +1,34 @@
...
         And I view the "silly-walks" group API "including" users
         Then I should see an element with xpath "//*[contains(string(), '"success": true,') and contains(string(), '"name": "group_admin"') and contains(string(), '"name": "walker"')]"
 
-    Scenario: Group membership is accessible to admins of the group
-        Given "Group Admin" as the persona
-        When I log in
-        And I view the "silly-walks" group API "including" users
-        Then I should see an element with xpath "//*[contains(string(), '"success": true,') and contains(string(), '"name": "group_admin"') and contains(string(), '"name": "walker"')]"
+        Examples: Admins
+            | User        |
```
Comment (duttonw) [inline test/features/groups.feature:11]: is user an admin?

Comment (ThrawnCA) [reply test/features/groups.feature:11]: The first 'example' is the field name to use.

File: test/features/organisations.feature [hunk 2]
```diff
@@ -1,53 +1,35 @@
...
         And I view the "department-of-health" organisation API "including" users
         Then I should see an element with xpath "//*[contains(string(), '"success": true,') and contains(string(), '"name": "organisation_admin"') and contains(string(), '"name": "editor"')]"
 
-    Scenario: Organisation membership is accessible to admins of the organisation
-        Given "Organisation Admin" as the persona
-        When I log in
-        And I view the "department-of-health" organisation API "including" users
-        Then I should see an element with xpath "//*[contains(string(), '"success": true,') and contains(string(), '"name": "organisation_admin"') and contains(string(), '"name": "editor"')]"
+        Examples: Admins
+            | User                |
```
Comment (duttonw) [inline test/features/organisations.feature:11]: User an admin?

**Review by duttonw — CHANGES_REQUESTED** [review]
review admin group as i don't think 'user' is meant to be in it.

File: test/features/organisations.feature (hunk 2 above)
Comment (ThrawnCA) [reply test/features/organisations.feature:11]: This is the variable name for the step to use.

File: test/features/groups.feature (hunk 1 above)
Comment (duttonw) [reply test/features/groups.feature:11]: ah, i see. tho may be better to call it 'admins' instead of user since don't we also have a 'user' persona?

Comment (ThrawnCA) [reply test/features/groups.feature:11]: How about 'Account'?

**Review by duttonw — APPROVED** [review]

**Review by duttonw — APPROVED** [review]

**Pull request merged**